Core.DomainModel/KendoConfig/KendoOrganizationalConfiguration.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Core.DomainModel.KendoConfig
+{
+    public class KendoOrganizationalConfiguration : Entity, IOwnedByOrganization
+    {
+        public KendoOrganizationalConfiguration()
+        {
+            VisibleColumns = new List<KendoColumnConfiguration>();
+        }
+
+        public OverviewType OverviewType { get; set; }
+        public string Version { get; set; }
+        public int OrganizationId { get; set; }
+        public Organization.Organization Organization { get; set; }
+        public virtual ICollection<KendoColumnConfiguration> VisibleColumns { get; set; }
+
+        public void AddColumns(IEnumerable<KendoColumnConfiguration> newColumns)
+        {
+            newColumns.ToList().ForEach(x =>
+            {
+                x.KendoOrganizationalConfigurationId = Id;
+                VisibleColumns.Add(x);
+            });
+
+
+            Version = ComputeVersion();
+        }
+
+        public static KendoOrganizationalConfiguration CreateConfiguration(int orgId, OverviewType overviewType)
+        {
+            return new ()
+            {
+                OrganizationId = orgId,
+                OverviewType = overviewType
+            };
+        }
+
+        private string ComputeVersion()
+        {
+            var visibleColumnsAsString = string.Join("", VisibleColumns.OrderBy(x => x.PersistId).Select(x => x.PersistId));
```

Comment:
i frontend laver du tolower og en specifik compare.. det gør du ikke her
som minimum bør du nok tolower her også for at få samme behavior